### PR TITLE
React Calculator: Milestone 1

### PR DESCRIPTION
**Pull request by RailonA**
- Use create-react-app to initialize a new React app
- Documentation: https://github.com/facebook/create-react-app
- Delete any unnecessary file or asset
- Add prop-types using npm for type checking props.
- Add big.js using npm.
- Run the application with npm start and make sure that everything works correctly.
- Follow the instructions in create-react-app-buildpack to deploy the app to Heroku

### CODE REVIEW

File: node_modules/.bin/acorn [hunk 1]
```diff
@@ -0,0 +1 @@
+../acorn/bin/acorn
```
Comment (rukundoeric) [inline node_modules/.bin/acorn]: - [x] <b>REQUIRED</b>: This file and any other files from `node_modules` directory, should not be committed. that means you should have put node_modules in `.gitignore`. Kindly check out this [Article](https://www.positronx.io/should-i-commit-the-node_modules-folder-to-git/) to learn more about why you should not commit `node_modules` files

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @RailonA , 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows gitwork-flow.  💯

## Required Changes ♻️
  - [x] Kindly deploy your app on Heroku, Netlify, or Firebase and make sure that the live-demo link is included in README file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
  - [x] Unnecessary files or assets, such as the React SVG Logo, should be deleted.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/)). Apparently in `/public` directory, there are still some useless images. Kindly remove them.
  - [x] Check the comments under the review form more...

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .gitignore [hunk 2]
```diff
@@ -21,3 +20,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+node_modules/
```
Comment (george-shammar) [inline .gitignore:24]: - [ ] As mentioned in the first review, please remove all unneccessary files from your repo. You still have node modules on github. Please note that even if you've added the node modules to .gitignore, you still need to remove the commited node modules from gitub.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @RailonA , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------

**Review by george-shammar — APPROVED** [review]
Hi @RailonA , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**